### ISSUE TITLE
Function "IsPluggedIn" does not accept XboxController enum.

### ISSUE THREAD

**Issue by rgero** [JISyed/Unity-XboxCtrlrInput#29]
In the latest released version, the function "IsPluggedIn" requires a parameter of type int. It would be nice if the function allowed for us to use the enum XboxController.

There is a work around where you can simply cast the enum to an int. You'd have to be careful about 
"XboxController.All" though.

**Comment by JISyed** [comment]
I was able to implement the method `IsPluggedIn()` to take in the enum `XboxController` as a parameter. The original method taking in an `int` still exists, but it's deprecated.

If `XboxController.Any` (or the deprecated `XboxController.All`) is passed in, XCI will check if any controllers are plugged in.

I released a [new version of XboxCtrlrInput](https://github.com/JISyed/Unity-XboxCtrlrInput/releases/tag/v1.8.0), which implements these changes. If everything works, Feel free to close this issue.
